packages/react-instantsearch-dom/src/index.js

```diff
@@ -1,5 +1,6 @@
 // Core
 export { createConnector } from 'react-instantsearch-core';
+export { HIGHLIGHT_TAGS } from 'react-instantsearch-core';
```

Comment:
you can do those in the same export with `,`

Reply:
Yep but I think it's a bit more easy to read with a separate statement.